<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ThresholdOptionsBase</class>
 <widget class="QWidget" name="ThresholdOptionsBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>308</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <widget class="QComboBox" name="typecomp" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="querycomp" />
   </item>
   <item row="4" column="1" >
    <widget class="QComboBox" name="badcomp" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_grade" >
     <property name="text" >
      <string>G&amp;rade:</string>
     </property>
     <property name="buddy" >
      <cstring>gradecomp</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_bcount" >
     <property name="text" >
      <string>&amp;Bad count:</string>
     </property>
     <property name="buddy" >
      <cstring>badcomp</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_qdate" >
     <property name="text" >
      <string>Last q&amp;uery:</string>
     </property>
     <property name="buddy" >
      <cstring>datecomp</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QComboBox" name="datecomp" />
   </item>
   <item row="4" column="2" >
    <widget class="QComboBox" name="badlist" />
   </item>
   <item row="3" column="2" >
    <widget class="QComboBox" name="querylist" />
   </item>
   <item row="5" column="2" >
    <widget class="QComboBox" name="datelist" />
   </item>
   <item row="1" column="2" >
    <widget class="QComboBox" name="typelist" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_type" >
     <property name="text" >
      <string>Word t&amp;ype:</string>
     </property>
     <property name="buddy" >
      <cstring>typecomp</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QComboBox" name="gradelist" />
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="gradecomp" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_qcount" >
     <property name="text" >
      <string>Query c&amp;ount:</string>
     </property>
     <property name="buddy" >
      <cstring>querycomp</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="2" column="1" >
      <widget class="QPushButton" name="b_all_less" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;All</string>
       </property>
      </widget>
     </item>
     <item rowspan="3" row="1" column="3" colspan="2" >
      <widget class="Q3ListBox" name="lessonlist" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>3</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="selectionMode" >
        <enum>Q3ListBox::Multi</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QPushButton" name="b_none_less" >
       <property name="text" >
        <string>&amp;None</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QLabel" name="l_lesson" >
       <property name="text" >
        <string>Selected lessons:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>50</height>
        </size>
       </property>
      </spacer>
     </item>
     <item rowspan="2" row="2" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4" >
      <widget class="QLabel" name="l_count" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_lesson" >
       <property name="text" >
        <string>&amp;Lesson:</string>
       </property>
       <property name="buddy" >
        <cstring>lessoncomp</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QComboBox" name="lessoncomp" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>290</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListBox</class>
   <extends></extends>
   <header>q3listbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
